### PR TITLE
fix(image) image creation, and export in custom projects

### PR DESCRIPTION
**Pull request by edlerd**
## Done

- fix(image) create image with alias in custom project
- fix(image) export image from a custom project

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - create an image with alias from an instance in a custom project
    - create an image with alias from an instance snapshot in a custom project
    - upload an image with alias to a custom project
    - export an image from a custom project

### CODE REVIEW

File: src/pages/instances/forms/CreateImageFromInstanceForm.tsx [hunk 1]
```diff
@@ -73,7 +73,14 @@ const CreateImageFromInstanceForm: FC<Props> = ({ instance, close }) => {
             (event) => {
               if (alias) {
                 const fingerprint = event.metadata.metadata?.fingerprint ?? "";
-                void createImageAlias(fingerprint, alias).then(notifySuccess);
+                void createImageAlias(fingerprint, alias, instance.project)
+                  .then(notifySuccess)
```
Comment (mas-who) [inline src/pages/instances/forms/CreateImageFromInstanceForm.tsx:77]: need to invalidate the query cache here

File: src/pages/instances/forms/CreateImageFromInstanceSnapshotForm.tsx [hunk 2]
```diff
@@ -82,7 +82,14 @@ const CreateImageFromInstanceSnapshotForm: FC<Props> = ({
             (event) => {
               if (alias) {
                 const fingerprint = event.metadata.metadata?.fingerprint ?? "";
-                void createImageAlias(fingerprint, alias).then(notifySuccess);
+                void createImageAlias(fingerprint, alias, instance.project)
+                  .then(notifySuccess)
```
Comment (mas-who) [inline src/pages/instances/forms/CreateImageFromInstanceSnapshotForm.tsx:86]: need to invalidate the query cache here

**Review by mas-who — CHANGES_REQUESTED** [review]
QA LGTM, small comment in code about query cache invalidation

**Comment by edlerd** [comment]
There are only two complicated things in programming ... something about cache. @mas-who please have another look

**Review by mas-who — APPROVED** [review]

**Comment by mas-who** [comment]
> There are only two complicated things in programming ... something about cache. @mas-who please have another look

Literally read that somewhere the other day haha

**Pull request merged**